lib/yard-ghpages.rb

```diff
@@ -23,9 +24,20 @@ def install(opts)
 
           desc 'Publish documentation to gh-pages'
           task :publish do |t|
+            source_branch = 'master'
+            source_directory = 'docs'
+            destination_branch = 'gh-pages' 
+
+            config_filename = '.yard-gh-pages.yml'
```

Comment:
I haven't used ruby or rake for a while, but does this work if someone calls rake from a subdirectory? IIRC rake will go to the parent directory until a Rakefile is found. I just want to make sure this always looks for the config file in the directory with the Rakefile, not the potential subdirectory that 'rake' is called from. Any idea if this works in that case?

Reply:
Yes, just tried this! When running from a subdirectory, it outputs what directory it's actually running in (in my case the root of the project)